chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -123,6 +123,8 @@
 
 machine_id_file = node[:tempest][:tempest_path] + '/machine.id'
 heat_machine_id_file = node[:tempest][:tempest_path] + '/heat_machine.id'
+insecure = ""
+insecure = "--insecure" if keystone_settings["insecure"]
```

Comment:
Well, it would need more work here, look at https://github.com/jsuchome/barclamp-tempest/blob/01ad0752c4126f628f259f3108a471f518f22573/chef/cookbooks/tempest/recipes/config.rb#L361